src/core/ext/lb_policy/round_robin/round_robin.c

```diff
@@ -269,7 +269,9 @@ static void rr_destroy(grpc_exec_ctx *exec_ctx, grpc_lb_policy *pol) {
   for (size_t i = 0; i < p->num_subchannels; i++) {
     subchannel_data *sd = p->subchannels[i];
     GRPC_SUBCHANNEL_UNREF(exec_ctx, sd->subchannel, "round_robin_destroy");
-    sd->user_data_vtable.destroy(sd->user_data);
+    if (sd->user_data_vtable != NULL) {
```

Comment:
I would check `sd->user_data` instead of `sd->user_data_vtable`.  The vtable is guaranteed to be non-NULL if the user_data is non-NULL (see https://github.com/grpc/grpc/blob/master/src/core/ext/client_channel/lb_policy_factory.c#L75), but what we really care about here is whether the user_data is non-NULL.